src/dataLayer/repositories/inmemory/items/index.spec.js

```diff
@@ -0,0 +1,36 @@
+import { ItemsRepo } from "."
```

Comment:
- We need to add setup and teardowns
- - More test case scanrios:
               - create item: 
                   - create item successfully. 
                   - fail item when all details not provided
                   - scenario when same request is run two time (it must generate two instances with unique ids)
               - getItems:
                   - get items returns empty array
                   - get items returns one item in result 
                   - get items return more than 1 item
               - deleteItem:
                   - deleting an item that does not exist
                   - deleting an item thatt exists